2dcontext/imagebitmap/createImageBitmap-origin.sub.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>createImageBitmap: origin-clean flag</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/media.js"></script>
+<script src="/common/namespaces.js"></script>
+<div id=log></div>
+<script>
+const crossOriginImageUrl = "http://{{domains[www1]}}:{{ports[http][0]}}/images/red.png";
+
+function assert_origin_unclean(bitmap) {
+  const context = document.createElement("canvas").getContext("2d");
+  context.drawImage(bitmap, 0, 0);
+  assert_throws("SecurityError", () => {
+    context.getImageData(0, 0, 1, 1);
+  });
+}
+
+function makeImage() {
+  return new Promise((resolve, reject) => {
+    const image = new Image();
+    image.onload = () => resolve(image);
+    image.onerror = reject;
+    image.src = crossOriginImageUrl;
+  });
+}
+
+const arguments = [
```

Comment:
@annevk @zcorpan if either of you have ideas of more cases that would be useful to test, please let me know.

Reply:
Maybe canvas backed by a worker canvas and such?